modules/install/pages/install.adoc

```diff
@@ -1,71 +1,538 @@
+_...output omitted..._
+Downloaded artifacts.
+JBoss modules installed.
+Configurations generated.
+JBoss examples installed.
+Server created in /opt/jboss-eap-installation-manager-1.1.6.Final-redhat-00001/bin/eap-8
+Operation completed in 541.33 seconds.
+----
+
+* Start the JBoss EAP standalone server using the start script present under `/opt/jboss-eap-8.0/bin/eap-8/bin/standalone.sh`.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server bin]$ sudo /opt/jboss-eap-8.0/bin/eap-8/bin/standalone.sh
+=========================================================================
+  JBoss Bootstrap Environment
+  JBOSS_HOME: /opt/jboss-eap-installation-manager-1.1.6.Final-redhat-00001/bin/eap-8
+  JAVA: java
+_...output omitted..._
+05:56:12,465 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0051: Admin console listening on http://127.0.0.1:9990
+05:56:12,488 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0025: JBoss EAP 8.0 Update 1.0 (WildFly Core 21.0.5.Final-redhat-00001) started in 3561ms - Started 282 of 524 services (318 services are lazy, passive or on-demand) - Server configuration file in use: standalone.xml
+----
+
+* Navigate to the JBoss EAP console at `http://localhost:8080` using firefox browser and confirm JBoss EAP is running as shown in the below screenshot.
++
+image::ui.png[align="center"]
+
+* Shutdown the JBoss EAP 8 server by pressing `Ctrl+C` in the terminal window where you started JBoss EAP 8.
+
+* To uninstall the JBoss EAP 8 standalone server, simply remove the `/opt/jboss-eap-8.0` directory.
+
+== Lab 3: Installing JBoss EAP using RPM method
+
+*Outcome*: In this lab, you subscribe your RHEL9 server and install JBoss EAP package. You configure the JBoss EAP server as a standalone server that can be access at `http://localhost:8080`.
+
+* Open a terminal instance and subscribe your RHEL9 server to the Red Hat Enterprise Linux Server base software repository using your `subscription-manager` credentials.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ sudo subscription-manager register --auto-attach
+Registering to: subscription.rhsm.redhat.com:443/subscription
+Username: XXXXX
+Password:XXXX
+----
+
+* Enable JBoss EAP 8 repository.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ sudo subscription-manager repos --enable=jb-eap-8.0-for-rhel-9-x86_64-rpms
+----
+
+* Install JBoss EAP 8.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ sudo dnf groupinstall jboss-eap8 -y
+----
+
+NOTE: The default `EAP_HOME` path for the RPM installation is `/opt/rh/eap8/root/usr/share/wildfly`.
+
+* Execute the JBoss EAP standalone server script present at `EAP_HOME/bin/standalone.sh` as a sudo user.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ /bin/sh /opt/rh/eap8/root/usr/share/wildfly/bin/standalone.sh -c standalone.xml -b 0.0.0.0
+----
+
+* Navigate to the JBoss EAP console at `http://localhost:8080` using firefox browser and confirm JBoss EAP is running as shown in the below screenshot.
```

Comment:
You have repeated the localhost:8080 home page thrice. Just do it once and mention in the step that "You should see the home page".

Reply:
Done the correction